object_detection/exporter_test.py

```diff
@@ -16,18 +16,16 @@
 """Tests for object_detection.export_inference_graph."""
 import os
 import numpy as np
-import six
 import tensorflow as tf
 from object_detection import exporter
 from object_detection.builders import model_builder
 from object_detection.core import model
 from object_detection.protos import pipeline_pb2
 
-if six.PY2:
-  import mock  # pylint: disable=g-import-not-at-top
```

Comment:
I am confused on this one...  All imports are now at the top of the file.  Does pylint complain?